lessons/students/1_basics/9_indexing_and_loops.py

```diff
@@ -38,12 +39,15 @@
 
 ####################################################################################################
 
+# TITLE: Section 2.1: String Indexing (30 seconds)
 # If you want to access a part of a string, you can get a portion of the string with indexing.
 beatles = "You say goodbye, and I say hello!"
 print("letters from the beatle song: " + beatles[3:11])
 
 ####################################################################################################
 
+# TITLE: Section 2.2: Combining String Indexing and Loops(10 min)
```

Comment:
Combining String Indexing and Loops(10 min) -> Combining List Indexing and Loops(10 min)